test/snippet/alphabet/quality/phred94.cpp

```diff
@@ -0,0 +1,15 @@
+#include <seqan3/alphabet/quality/phred94.hpp>
+#include <seqan3/core/debug_stream.hpp>
+
+int main()
+{
+    seqan3::phred94 phred;
+    phred.assign_rank(2); // wrapper for assign_phred(2)
+    seqan3::debug_stream << static_cast<int>(phred.to_phred()) << "\n"; // 2
+    seqan3::debug_stream << phred.to_char() << "\n";        // '#'
+    seqan3::debug_stream << static_cast<int>(phred.to_rank()) << "\n";  // 2
+
+    seqan3::phred94 another_phred{75};
+    seqan3::debug_stream << another_phred.to_phred() << "\n"; // 75
+    // we need to cast to (int) for human readable console output
```

Comment:
But we don't cast here to int? Also, if you write a complete sentence in a comment, I would use a dot in the end and capitalize the first word.